stories/data/overlord-banner.js

```diff
@@ -0,0 +1,326 @@
+const banner = [
+  {
+    'Actions': 0,
+    'Bids': 21,
+    'Clicks': 0,
+    'ImpCost': 0.0452,
+    'Imps': 18,
+    'Revenue': 0.171,
+    'BannerCode': 110359,
+    'Date': '01-03-2020',
+    'Name': '300x250-TacoTime_Burrito_Bowl_with_Free_Pop_2',
```

Comment:
I would change 'TacoTime_Burrito_Bowl_with_Free_Pop' with 'Meal_with_Pop' so we don't make public data about companies we deal with. And erase this file from github ?!

Reply:
done